src/components/container/SeverityCalendar/SeverityCalendar.tsx

```diff
@@ -29,9 +29,11 @@ interface SeverityLogEntry {
 
 export default function (props: SeverityCalendarProps) {
     const surveyAnswerQuery: SurveyAnswersQuery = {
-        surveyName: props.surveyName,
         resultIdentifier: props.dateRecordedResultIdentifier ? [props.dateRecordedResultIdentifier, props.severityResultIdentifier] : [props.severityResultIdentifier]
     };
+    if (props.surveyName) {
+        surveyAnswerQuery.surveyName = props.surveyName;
```

Comment:
This conditionally setting the surveyName rather than just setting it to undefined is because MyDataHelps.JS has some weird behavior around defined *keys* with undefined values, so we don't want to set the key at all